user-service/src/main/java/rs/edu/raf/banka1/requests/order/CreateOrderRequest.java

```diff
@@ -0,0 +1,17 @@
+package rs.edu.raf.banka1.requests.order;
+
+import lombok.Getter;
+import lombok.Setter;
+import rs.edu.raf.banka1.model.OrderType;
+
+@Getter
+@Setter
+public class CreateOrderRequest {
+    private OrderType orderType;
+    private Long stockId;
+    private Long userId;
```

Comment:
Ovo će nam se najverovatnije zvati customerId, pošto smo radili refaktorisanje koda i više nemamo `User` tabelu.

Reply:
zar nije agentId

Reply:
ako je ko pravi onda `agentId`, ako je za koga se pravi, onda `customerId`